client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,5 @@
+<div class="msg {{type}}{{#if self}} self{{/if}}{{#if highlight}} highlight{{/if}} closed" id="msg-{{id}}">
+	<span class="time"></span>
+	<span class="from"><span class="expander"></span></span>
+	<span class="text">Condensed joins/parts</span>
```

Comment:
Would you be happy with me just saying "Condensed joins/parts/mode/nick changes" for this PR?